### DISCUSSION TITLE
App/Deep link support?

### DISCUSSION THREAD

**Discussion by arnolicious** [jellyfin/jellyfin-android#1329] [General]
Hi everyone,
I was wondering if deep links (app links) are currently supported or if that would be a good feature to have.
For example being able to launch the app and point to a specific movie or episode.

This would be useful for any companion/media management type of apps, that could directly link the user to a movie in the jellyfin app.

Cheers!

**Comment by nielsvanvelzen** [comment]
https://github.com/jellyfin/jellyfin-meta/discussions/68
